features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -41,7 +41,7 @@ export function SidebarNavigation() {
           {/* eslint-disable-next-line @next/next/no-img-element */}
           <img
             src={
-              isSidebarCollapsed
+              isSidebarCollapsed && window.innerWidth > 1024
```

Comment:
This is a simple solution that works fine except when the user changes the window size or rotates the mobile device. The problem is that the size change of the browser window doesn't trigger a re-render of the React app and thus this code wouldn't execute. Taking that into account alternative approaches as in your other PR work better.